rust-toolchain

```diff
@@ -1 +1 @@
-1.49.0
+1.50.0
```

Comment:
If we truly have to upgrade toolchain. It is fine to me. But better that we can add a separated patch doing so.

Reply:
Without upgrading the toolchain, this path can not pass compiling , is that acceptable?

Reply:
Agree